ci/lib.sh

```diff
@@ -57,7 +57,17 @@ curl() {
 # This will contain the artifacts we want.
 # https://developer.github.com/v3/actions/workflow-runs/#list-workflow-runs
 get_artifacts_url() {
-  curl -fsSL 'https://api.github.com/repos/cdr/code-server/actions/workflows/ci.yaml/runs?status=success&event=push' | jq -r ".workflow_runs[] | select(.head_sha == \"$(git rev-parse HEAD)\") | .artifacts_url" | head -n 1
+  local head_sha
+  local artifacts_url
+  head_sha=$(git rev-parse HEAD)
+  artifacts_url=$(curl -fsSL 'https://api.github.com/repos/cdr/code-server/actions/workflows/ci.yaml/runs?status=success&event=pull_request' | jq -r ".workflow_runs[] | select(.head_sha == \"$head_sha\") | .artifacts_url" | head -n 1)
+  if [[ -z "$artifacts_url" ]]; then
+    echo "ERROR: artifacts_url came back empty"
+    echo "Check the get_artifacts_url function"
```

Comment:
I think this will not show up because the stdout of this function is used directly in the curl command. We could log it to stderr.

```suggestion
    >&2 echo "ERROR: artifacts_url came back empty"
    >&2 echo "Check the get_artifacts_url function"
```

Reply:
Nice catch! TIL!